addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -861,8 +867,9 @@ def _l10n_it_post_invoices_step_1(self, invoices):
 
             if invoice._is_commercial_partner_pa():
                 invoice.message_post(
-                    body=(_("Invoices for PA are not managed by Odoo, you can download the document and send it on your own."))
-                )
+                    body=(_("Odoo doesn't manage invoices involving Public Administration partners."
```

Comment:
Here we needed a little more clarity instead. "Download the document and send it on your own" doesn't help the user much.